apps/backend/db_patches/0183_RemoveUserColulmns2.sql

```diff
@@ -0,0 +1,24 @@
+-- 0181_RemoveUserColumns.sql
+DO
+$$
+BEGIN
+    IF register_patch(
+        '0182_RemoveUserColumns2.sql',
```

Comment:
suggestion: No need for the numbers, they might go easily out of sync with the file name
```suggestion
        'RemoveUserColumns2.sql',
```